scripts/io_helper.py

```diff
@@ -0,0 +1,4 @@
+def write_to_file(filepath, content):
```

Comment:
Returning a boolean flag on status of writing file would be great

Reply:
Sure bro :)

Reply:
Done @vivonk

Reply:
Done @vivonk

Reply:
Can you review it again? @vivonk